tests/rules/test_python_command.py

```diff
@@ -0,0 +1,9 @@
+from thefuck.main import Command
+from thefuck.rules.sudo import match, get_new_command
```

Comment:
Looks like the wrong import... Shouldn't you import `thefuck.rules.python_command` here?